railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -46,6 +50,7 @@ def self.plugin_rails_init(options)
 
   mattr_accessor(:run_with_autorun)         { false }
   mattr_accessor(:run_with_rails_extension) { false }
+  mattr_accessor(:test_patterns) { nil }
```

Comment:
Let's get rid of the test patterns and add a `rake_run` method, that looks like:

```
# Running several Rake tasks in a single command would trip up the runner,
# as the patterns would also contain the other Rake tasks.
def self.rake_run(patterns) # :nodoc:
  @rake_patterns = patterns
  run
end

def self.plugin_rails_init(options)
  # Other code

  unless run_with_autorun
    ::Rails::TestRequirer.require_files @rake_patterns || options[:patterns]
  end

  # More initialization...
end
```